### DISCUSSION TITLE
There is no sum to 8 option?

### DISCUSSION THREAD

**Discussion by AstartesFanboy** [chummer5a/chummer5a#4660] [Q&A] [answered]
So, uh. Im using the latest stable release and I cant do sum to 8? Why is that? It should be a viable choice but the only choice is a sum to 10 and the other non sum ones. Anybody know how to fix this?

**Comment by ArchonMegalon** [comment]
Is Sum to 8 in any way official RAW?

**Comment by DelnarErsike** [comment, marked as answer]
In case are still tracking this, you can create your own character settings within Chummer using the character settings editor. Within that editor, if you choose the Sum-to-Ten build method, you will see a field where you can set what the sum should be (default is, of course, 10).
